packages/image-minify/index.js

```diff
@@ -1,49 +1,26 @@
 const merge = require('deepmerge');
-const ImageminWebpackPlugin = require('imagemin-webpack/ImageminWebpackPlugin');
-const gifsicle = require('imagemin-gifsicle');
-const svgo = require('imagemin-svgo');
-const pngquant = require('imagemin-pngquant');
-const mozjpeg = require('imagemin-mozjpeg');
+const ImageminWebpackPlugin = require('imagemin-webpack-plugin').default;
 const webp = require('imagemin-webp');
 
-const imageminLoader = require.resolve('imagemin-webpack/imagemin-loader');
-
 module.exports = (neutrino, opts = {}) => {
   const options = merge({
     imagemin: {
-      plugins: [
-        gifsicle(),
-        svgo(),
-        pngquant(),
-        mozjpeg(),
-        webp()
-      ]
-    },
-    plugin: {
-      name: '[path][name].[ext]'
+      plugins: [],
+      optipng: {},
+      gifsicle: {},
+      jpegtran: {},
+      svgo: {},
+      pngquant: null,
+      webp: {}
     },
-    pluginId: 'imagemin',
-    useId: 'imagemin',
-    rules: ['svg', 'img']
+    pluginId: 'imagemin'
   }, opts);
 
-  const test = options.rules.map((ruleId) => {
-    neutrino.config.module
-      .rule(ruleId)
-      .use(options.useId)
-        .loader(imageminLoader)
-        .options(options.imagemin);
-
-    return neutrino.config.module.rule(ruleId).get('test');
-  })
-  .filter(Boolean);
-
-  options.plugin = merge({
-    imageminOptions: options.imagemin,
-    test
-  }, options.plugin);
+  if (options.webp) {
+    options.imagemin.plugins.push(webp(options.webp));
+  }
 
```

Comment:
I wonder how we should decide what to include by default…it would seem that `imagemin-webpack-plugin` enables "optimizers" by default, but not "compressors". Including `imagemin-webp` by default may be going against that.

However if we do, perhaps we should do the same with `imagemin-mozjpeg`. That is included in the current version of of this MW, but not present in this PR (nor is any jpeg compression).